examples/text_classification/ios/TextClassifierTests/TextClassifierTests.swift

```diff
@@ -0,0 +1,130 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+@testable import TextClassifier
+import MediaPipeTasksText
+
+final class TextClassifierTests: XCTestCase {
+
+    static let bertModel = Model.mobileBert
+
+    static let testText = "unflinchingly bleak and desperate"
```

Comment:
Well that's definitely a negative text block. Are you doing ok? :p

Reply:
This text is in https://github.com/google/mediapipe/blob/master/mediapipe/tasks/ios/test/text/text_classifier/TextClassifierTests.swift
only negativeText is used.
I will change the variable to negativeText.